wifiphisher/pywifiphisher.py

```diff
@@ -510,128 +514,6 @@ def dhcp(dhcpconf, mon_iface):
     return True
 
 
-# Wifi Jammer stuff
-# TODO: Merge this with the other channel_hop method.
-def channel_hop2(mon_iface):
-    '''
-    First time it runs through the channels it stays on each channel for
-    5 seconds in order to populate the deauth list nicely.
-    After that it goes as fast as it can
-    '''
-    global monchannel, first_pass, args, jamming_daemon_running
-
-    channelNum = 0
-
-    if args.channel:
-        with lock:
-            monchannel = args.channel
-    while jamming_daemon_running:
-        if not args.channel:
-            channelNum += 1
-            if channelNum > 11:
-                channelNum = 1
-                with lock:
-                    first_pass = 0
-            with lock:
-                monchannel = channelNum
-            mon_iface.set_channel(monchannel)
```

Comment:
I don't see anywhere in the new code where you set the channel of the interface to the channel of the target Access Point.